bebop_driver/launch/bebop_node.launch

```diff
@@ -10,6 +10,6 @@
             <param name="bebop_ip" value="$(arg ip)" />
             <rosparam command="load" file="$(arg config_file)" />
         </node>
-        <include file="$(find bebop_description)/launch/description.launch" />
+        <include file="$(find bebop_description)/launch/description.launch">
```

Comment:
Thanks for re-enabling this. However this will not run since it's not a valid XML. Please also add back the missing ` />`. It might be easier to just checkout `bode_node.launch` from upstream.